pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsebscsidriver
```

Comment:
This goes against the [Go package naming conventions](https://blog.golang.org/package-names). I suggest we manage this component under the following Go path: `pkg/components/csi/aws`

Reply:
This would be different from how other components are named and stored. Should those be changed?

Reply:
Let's revisit this later.